examples/schema_template_vrf/main.tf

```diff
@@ -11,13 +11,77 @@ provider "mso" {
   password = "" # <MSO pwd>
   url      = "" # <MSO URL>
   insecure = true
+  platform = "nd"
 }
 
-resource "mso_schema_template_vrf" "demovrf01" {
-    schema_id       = "5c4d5bb72700000401f80948"
-    template        ="Temp200"
-	name            = "vrf982"
-	display_name    ="vrf982"
-	layer3_multicast=false
-  
+data "mso_tenant" "tenant_test" {
+  name         = "ansible_test"
+  display_name = "ansible_test"
+}
+
+resource "mso_schema" "schema_test" {
+  name = "terraform_schema"
+  template {
```

Comment:
```
template {
    name         = "Template1"
    display_name = "Template1"
    tenant_id    = data.mso_tenant.tenant_test.id
  }
  template {
    name         = "Template2"
    display_name = "Template2"
    tenant_id    = data.mso_tenant.tenant_test.id
  }
  template {
    name         = "Template3"
    display_name = "Template3"
    tenant_id    = data.mso_tenant.tenant_test.id
  }
```

Reply:
Why should there be 3 templates?

Reply:
It's more intuitive to have more than one Template here because we're accessing the first Template in the next resource via referencing. It would make it clearer to the user.

Reply:
Since, template 2 and template 3 is not required in further test cases, it will not be added to schema.